doc/challenge/basic/find_specific_information.md

```diff
@@ -0,0 +1,69 @@
+# find specific information
```

Comment:
This file is linked nowhere. The information of this document need to be organized and put somewhere.